tabpy-server/tabpy_server/tabpy.py

```diff
@@ -865,7 +849,7 @@ def set_parameter(settings_key, config_key, default_val=None, check_env_var=Fals
     # if state.ini does not exist try and create it - remove last dependence
     # on batch/shell script
     state_file_path = settings['state_file_path']
-    logger.info("Loading state from state file {}/state.ini".format(state_file_path))
+    logger.info("Loading state from state file {}".format(os.path.join(state_file_path, "state.ini")))
```

Comment:
Secondary to this PR but should consider using pathlib operations instead of `os.path` going forward

Reply:
Will fix in the future... :)